packages/storybook/stories/va-link.stories.jsx

```diff
@@ -153,14 +159,38 @@ Calendar.args = {
   text: `Add to calendar`,
 };
 
+export const Icon = VariantTemplate.bind(null);
+Icon.args = {
+  ...defaultArgs,
+  'href': '#',
+  'text': `Link with icon`,
+  'icon-name': 'chevron_right',
```

Comment:
Is `chevron_right` a realistic use case? I'm not sure what icons were requested originally. Maybe social media icons? One of those might be a better example.

Reply:
I couldn't find any specific examples yet. Do you have any examples or suggestions I can refer to?

Reply:
hmmm 🤔...  I'm not sure. On the [bottom right of this page,](https://www.va.gov/burials-memorials/) I see `va-link` trying to use `mail` and `facebook` icons. Maybe one of those would be good?

Reply:
Yes, I'll include the mail icon.